autogpt_platform/backend/backend/data/execution.py

```diff
@@ -588,11 +588,11 @@ async def update_graph_execution_start_time(
 
 async def update_graph_execution_stats(
     graph_exec_id: str,
-    status: ExecutionStatus,
+    status: ExecutionStatus | None = None,
     stats: GraphExecutionStats | None = None,
 ) -> GraphExecution | None:
     update_data: AgentGraphExecutionUpdateManyMutationInput = {
-        "executionStatus": status
+        "updatedAt": datetime.now(tz=timezone.utc),
```

Comment:
I feel like the DB should be doing this automatically/implicitly, doesn't it?

Reply:
Good point, I think so